### PR TITLE
SRE Weekly posts

### PR DESCRIPTION
**Pull request by JensRantil**

### CODE REVIEW

**Comment by BigEd** [comment]
Those look interesting to me, although I haven't clicked through yet. I notice some typos:
"balancer lead to lead to increased" -> "balancer lead to increased"
"flouride" -. "fluoride"
"rately modified" -> "rarely modified"
"proxied traffix to unintentional end users" -> "proxied traffic to unintended end users"
"scascading error where fronend" -> "cascading error where frontend"
"lead to customers experienced write" -> "lead to customers experiencing write"
"outage due AWS" -> "outage due to AWS"

File: README.md [hunk 1]
```diff
@@ -172,8 +192,12 @@ a migration of a critical backend system.
 
 [Sentry](http://blog.getsentry.com/2015/07/23/transaction-id-wraparound-in-postgres.html). Transaction ID Wraparound in Postgres caused Sentry to go down for most of a working day.
 
+[Skyliner](https://blog.skyliner.io/post-mortem-outages-on-1-19-17-and-1-23-17-3f65cc6f693e). A memory leak in a third party library lead to Skyliner being unavailable for 4 days(!).
```
Comment (ThomasWo) [inline README.md]: Reading the linked post, it sounded more like the 2 incidents, while having the same root cause, were separate instances with a 4 day gap of regular service in between.

**Comment by JensRantil** [comment]
Thanks for proofreading this, guys. Clearly I must have been drunk when I submitted this, given the amount typos and incorrect statement. 😂  Looks better now?

**Comment by ThomasWo** [comment]
Lgtm

**Comment by BigEd** [comment]
Looks good to me - @danluu perhaps you can merge before too many conflicts crop up?

**Pull request closed without merging**